src/screens/InitScreen.js

```diff
@@ -62,8 +63,9 @@ const InitScreen = () => {
   };
   return (
     <>
-      <Header title="Payments" onBackClick={gotoHome} />
+      <Header onBackClick={gotoHome} />
```

Comment:
think about some encapsulation in next MR

Reply:
okay